src/mono/wasm/runtime/gc-handles.ts

```diff
@@ -109,11 +109,15 @@ export function teardown_managed_proxy(result: any, gc_handle: GCHandle): void {
 
 export function assert_not_disposed(result: any): GCHandle {
     const gc_handle = result[js_owned_gc_handle_symbol];
-    mono_assert(gc_handle != GCHandleNull, "ObjectDisposedException");
+    mono_check(gc_handle != GCHandleNull, "ObjectDisposedException");
```

Comment:
i don't see mono_check imported or defined here

Reply:
`mono_assert` global macro was renamed to `mono_check`. The new `mono_assert` is actual function which will abort the runtime.